playbooks/flink-end-to-end-test/build_and_test_core.yaml

```diff
@@ -0,0 +1,54 @@
+- import_playbook: common.yaml
+
+# NOTE: Some tests should not be ran under root.
+- hosts: all
+  become: no
```

Comment:
if non root users, the following `apt ...` command should be added `sudo` prefix ?